src/components/ObservationsFlashList/ObsListItem.js

```diff
@@ -33,7 +33,8 @@ type Props = {
   uploadProgress?: number,
   unsynced: boolean,
   hideObsUploadStatus?: boolean,
-  hideObsStatus?: boolean
+  hideObsStatus?: boolean,
+  simpleObsStatus?: boolean
```

Comment:
since `simpleObsStatus` is a boolean, can we use one of the naming conventions for booleans? it's generally best to name them in a way that conveys true/false, like `isSimpleObsStatus`, `showSimpleObsStatus`, `displaySimpleObsStatus`.